frontend/src/app/containers/ExperimentPage/ExperimentButtons.js

```diff
@@ -0,0 +1,241 @@
+import React from "react";
```

Comment:
This entire file is basically just extracted button types from the `ExperimentControls` Page